doc/tutorials/dnn/dnn_halide/dnn_halide.markdown

```diff
@@ -8,20 +8,7 @@ according to specific device and evaluate it with a quite good efficiency.
 
 An official website of the Halide project: http://halide-lang.org/.
 
-## Efficiency comparison
-Measured on Intel&reg; Core&trade; i7-6700K CPU @ 4.00GHz x 8.
-
-Single image forward pass (in milliseconds):
-
-|     Architecture | MKL backend | Halide backend | Speed Up ratio |
```

Comment:
what is exactly meant by MKL backend here? I think, instead of putting the table (that can become obsolete rather quickly) into the tutorial, let's just put the link to our wiki with all the measurements.

Reply:
You're right. This patch removes the old table that was added initially.